dashboard/test/serializers/api/v1/pd/enrollment_flat_attendance_serializer_test.rb

```diff
@@ -178,7 +178,13 @@ class Api::V1::Pd::EnrollmentFlatAttendanceSerializerTest < ::ActionController::
   end
 
   test 'extract school and teacher info when they are empty' do
-    enrollment = build :pd_enrollment, school_info: (build :school_info_us)
+    enrollment = build(
+      :pd_enrollment,
+      # Don't persist school info; it's too empty to pass validation.
```

Comment:
This test is explicitly calling `build :school_info_us` rather than just using the default `school_info` factory because the default factory also creates associated school data and other stuff that's required for the model to pass validation, but this test explicitly wants to check the serializer behavior when that stuff is missing. So, `build` instead of `create` so we don't even try to apply the validation logic.

Reply:
Ah yes, makes sense. Thank you!